descope/management/common.py

```diff
@@ -18,3 +18,9 @@ class MgmtV1:
 
     # jwt
     updateJwt = "/v1/mgmt/jwt/update"
+
+    # permission
+    permissionCreatePath = "/v1/mgmt/permission/create"
+    permissionUpdatePath = "/v1/mgmt/permission/update"
+    permissionDeletePath = "/v1/mgmt/permission/delete"
+    permissionLoadAllPath = "/v1/mgmt/permission/all"
```

Comment:
"all" ?? not sounds good for an api endpoint..
maybe something like "list"/"load" (or im sure you can find some word better then me.. :) )